src/main/java/com/googleinterns/zoomtube/servlets/DiscussionServlet.java

```diff
@@ -85,20 +88,23 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     final Entity commentEntity;
     long lectureId = Long.parseLong(request.getParameter(PARAM_LECTURE));
-    Key lecture = KeyFactory.createKey(LectureUtil.KIND, lectureId);
-    // TODO: Require non-zero content length. See: #183.
+    Key lectureKey = KeyFactory.createKey(LectureUtil.KIND, lectureId);
+    // TODO: Validate that the transcript line contains the timestampMs.
+    long transcriptId = Long.parseLong(request.getParameter(PARAM_TRANSCRIPT_LINE));
+    Key transcriptLineKey = KeyFactory.createKey(TranscriptLineUtil.KIND, transcriptId);
```

Comment:
If the lecture has no transcript, then does this still returns a transcriptId?

Reply:
talked to amy about how we want to handle this, and had decided to make this param optional